### ISSUE TITLE
Format and coding style of SPARQL Test Queries

### ISSUE THREAD

**Issue by kuzeko** [BONSAMURAIS/triple-store#7]
Test queries should :

- [x] define prefix
- [x] user the .rq [file extension](https://www.w3.org/2001/sw/DataAccess/rq23/#mediaType)
- [x] contain in comment the  natural language description of the task they address
- [x] DONT contain the named graph in the `FROM` clause (see  [W3C document](https://www.w3.org/2001/sw~~/DataAccess/rq23/#specifyingDataset) ) : _syntax doesn't work in Fuseki_

**Comment by mmr2187** [comment]
acknowledged. I am only doing the preliminary queries to identify holes in the ontology if any. Hopefully, there are none.

**Comment by mmr2187** [comment]
essentially, all queries in the current query folder need to have the following (as necessary) included

```
PREFIX owl: <http://www.w3.org/2002/07/owl#> .
PREFIX dc: <http://purl.org/dc/terms/> .
PREFIX ns0: <http://purl.org/vocab/vann/> .
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
PREFIX foaf: <http://xmlns.com/foaf/0.1/> .
PREFIX cc: <http://creativecommons.org/ns#> .
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#> .
PREFIX skos: <http://www.w3.org/2004/02/skos/core#> .
PREFIX schema: <http://schema.org/> .
PREFIX bont: <http://ontology.bonsai.uno/core#> .
PREFIX om2: <http://www.ontology-of-units-of-measure.org/resource/om-2/> .
PREFIX time: <http://www.w3.org/2006/time#> .
```

and possibly the other namespaces we used. And I haven't included those yet as I'm simply testing out the logic.

**Comment by tmillross** [comment]
I'm fixing the file suffix and comment issues now

**Comment by kuzeko** [comment]
Attention, the use of `< >` is mandatory

e.g. `PREFIX  xsd:    <http://www.w3.org/2001/XMLSchema#>`

See

https://jena.apache.org/tutorials/sparql_datasets.html

**Comment by mmr2187** [comment]
Right. I did that, it just put things as hyperlinks. forgot about markdown. But I did forget to remove the @
